package.json

```diff
@@ -11,16 +11,16 @@
   "dependencies": {},
   "devDependencies": {
     "@slack/client": "^3.12.0",
-    "husky": "^0.14.3",
-    "lint-staged": "^6.0.0",
     "@types/jest": "^20.0.8",
     "@types/node": "^8.0.25",
-    "danger": "^2.0.0-alpha.8",
+    "danger": "^3.1.0",
     "danger-plugin-spellcheck": "^0.7.7",
     "danger-plugin-yarn": "^0.3.2",
     "github-webhook-event-types": "^1.0.0",
+    "husky": "^0.14.3",
     "jest": "^21.0.2",
-    "prettier": "^1.9.2",
+    "lint-staged": "^6.0.0",
+    "prettier": "^1.10.2",
     "ts-jest": "^21.0.0",
     "typescript": "^2.3.2"
```

Comment:
Most of these dependencies are added so that the typing in this repo is correct, so updating them is no biggie.